package.json

```diff
@@ -216,7 +214,6 @@
     "jest-junit": "16.0.0",
     "jest-matcher-utils": "29.3.1",
     "lerna": "5.5.4",
-    "lint-staged": "13.2.3",
```

Comment:
now this is removed, there's an entire `lint-staged` section further down in the `package.json` that can be removed as well